### ISSUE TITLE
Reproduciple chrash on playing an .mkv video

### ISSUE THREAD

**Issue by wirepatch** [nextcloud/android#7388]
### Steps to reproduce
1. Play a video starting in Nextcloud client GUI
2. Wait ~10 seconds
3. 

### Expected behaviour
- Tell us what should happen
The video should continue playing
### Actual behaviour
- Tell us what happens
Crash of Nextcloud client providing a stack trace
### Can you reproduce this problem on https://try.nextcloud.com?
Don't know
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:
10
Device model: 
Galaxy tab S5e
Stock or customized system:

Nextcloud app version:
Current
Nextcloud server version:
11
Reverse proxy:
Nginx
#### Web server error log
```
************ CAUSE OF ERROR ************

android.app.RemoteServiceException: Context.startForegroundService() did not then call Service.startForeground(): ServiceRecord{ab71040 u0 com.nextcloud.client/.media.PlayerService}
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2188)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:237)
	at android.app.ActivityThread.main(ActivityThread.java:8167)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:496)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1100)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30140090
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: samsung
Device: gts4lvwifi
Model: SM-T720
Id: QP1A.190711.020
Product: gts4lvwifieea

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: T720XXU1CTI1

```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud client log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by aqaqsubin** [comment]
I think it's settled in #7340

**Comment by wirepatch** [comment]
When will this fix be publicly available by Google Play store? Is there a »pre« version for manual pkg update on offer? Using the most current version the error still persists.

**Comment by aqaqsubin** [comment]
@wirepatch 
You can know after PR #7340 is merged.
PR has not yet been merged into the latest release.
If you want to check whether the error be fixed or not, go to #7340 and download the apk.

**Comment by wirepatch** [comment]
Confirmation of successful fix provided by 7340.apk. Thx!
